chrome-app/links.js

```diff
@@ -36,7 +36,7 @@ const LINKS = [
     },
     {
       title: "Shepherd",
-      href: gitHubQuery("https://github.com/pulls", [...pr, "assignee:USERNAME", "-author:USERNAME" ]),
+      href: gitHubQuery("https://github.com/pulls", [...pr, "assignee:USERNAME", "-author:USERNAME", "review:required" ]),
```

Comment:
What does this do? Ignore drafts?

Reply:
remove PRs that are in "changes requested" and "approved" from this list. Show me only the stuff that's waiting for my review (nirvana!)